gc/startup/omrgcalloc.cpp

```diff
@@ -60,8 +60,13 @@ allocHelper(OMR_VMThread * omrVMThread, size_t sizeInBytes, uintptr_t flags, boo
 
 	omrobjectptr_t heapBytes = (omrobjectptr_t)env->_objectAllocationInterface->allocateObject(env, &allocdescription, env->getMemorySpace(), collectOnFailure);
 
-	/* OMRTODO: Should we use zero TLH instead of memset? */
 	if (NULL != heapBytes) {
+#if defined(OMR_GC_ALLOCATION_TAX)
```

Comment:
The currently allocated object does not appear to be reachable.  I think it could be lost during the payAllocationTax call.  Also even if the object was reachable during the GC it is not initialized which may cause the GC to crash